modules/dxp/apps/commerce/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/diagram_type/default.jsp

```diff
@@ -16,14 +16,4 @@
 
 <%@ include file="/init.jsp" %>
 
-<%
-CSDiagramSettingDisplayContext csDiagramSettingDisplayContext = (CSDiagramSettingDisplayContext)request.getAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT);
-
-CPDefinition cpDefinition = csDiagramSettingDisplayContext.getCPDefinition();
-
-String type = DefaultCSDiagramType.KEY;
-
-CSDiagramSetting csDiagramSetting = csDiagramSettingDisplayContext.fetchCSDiagramSetting();
-%>
-
-<%@ include file="/diagram_type/diagram.jspf" %>
\ No newline at end of file
+<div></div>
```

Comment:
Good point! I did it on purpose just to keep in mind its role as a component wrapper. It will contain some stuffs in the next pr which will arrive soon and it passed sf so let's keep it pls :)

Reply:
Sure thing!